client-gen/tests/snapshots/generator__rmrk_works.snap

```diff
@@ -146,90 +146,190 @@ where
     }
 }
 
-#[derive(Default)]
-pub struct RmrkCatalogCallBuilder;
+pub mod rmrk_catalog_calls {
```

Comment:
I would suggest naming it just `io` - it is what they are used to now + generated client is already in a mod like `puppet` so `puppet::io` is quite intuitive

Reply:
renamed